server/routerlicious/.changeset/dirty-keys-follow.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/server-lambdas": major
+"@fluidframework/server-routerlicious": major
+"@fluidframework/server-routerlicious-base": major
+"@fluidframework/server-services-shared": major
```

Comment:
I think this specific changeset can be scoped to `services-shared` and be considered `minor`